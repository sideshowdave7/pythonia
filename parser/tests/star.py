*a, = range(5)
seq = range(5)
a, *b, c = seq

