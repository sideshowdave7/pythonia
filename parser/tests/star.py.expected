(program
 (= ((star a)) (indexed range (called 5)))
 (= (seq) (indexed range (called 5)))
 (= (a (star b) c) seq))

